model-api/src/main/java/edu/unc/lib/boxc/model/api/exceptions/TombstoneFoundException.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.model.api.exceptions;
+
+/**
+ * Exception indicating that the requested resource has been destroyed and replaced by a tombstone object
+ *
+ * @author bbpennel
+ */
+public class TombstoneFoundException extends RepositoryException {
+    private static final long serialVersionUID = 1L;
```

Comment:
Its not, but intellij gives a warning without it. Its because the class inherits from Exception, which ultimately inherits from Serializable, which expects a serialVersionUID to be present

Reply:
Huh